src/main/default/lwc/descriptionInput/descriptionInput.js

```diff
@@ -1,45 +1,60 @@
+/* eslint-disable no-undef */
 import { LightningElement, api } from 'lwc';
 import explainProblem from '@salesforce/label/c.cookbook_DescriptionInputTitle';
 import errorValueMissing from '@salesforce/label/c.cookbook_ValueMissing';
+import {
+  registerComponentForInit,
+  initializeWithHeadless
+} from 'c/quanticHeadlessLoader';
+import { debounce } from 'c/utils';
+
+/** @typedef {import("coveo").CaseAssistEngine} CaseAssistEngine */
+/** @typedef {import("coveo").CaseInput} CaseInput */
 
 /**
  * The `descriptionInput` component displays a rich text input for the case description.
  * @example
- * <c-description-input label="Explain the problem" messageWhenValueMissing="Complete this field." required></c-description-input>
+ * <c-description-input engineId={engineId} caseEditDelayMs="500" label="Explain the problem" messageWhenValueMissing="Complete this field." required displayStrengthIndicator></c-description-input>
```

Comment:
BTW @mmitiche all the attributes, defined with `lowerUpper` in your component map to `lower-upper` in the HTML when you use it.

Reply:
```suggestion
 * <c-description-input engine-id={engineId} case-edit-delay-ms="500" label="Explain the problem" message-when-value-missing="Complete this field." required display-strength-indicator></c-description-input>
```